recipes-security/optee-imx/optee-os_3.10.0.imx.bb

```diff
@@ -51,30 +50,25 @@ EXTRA_OEMAKE = " \
 	PLATFORM_FLAVOR=${PLATFORM_FLAVOR} \
 	CROSS_COMPILE=${HOST_PREFIX} \
 	CROSS_COMPILE64=${HOST_PREFIX} \
-	LDFLAGS= \
-	O=${B} \
+	CFG_TEE_TA_LOG_LEVEL=0 \
+	CFG_TEE_CORE_LOG_LEVEL=0 \
+	-C ${S} O=${B}\
 "
 
-do_compile () {
-    unset LDFLAGS
-    export CFLAGS="${CFLAGS} --sysroot=${STAGING_DIR_HOST}"
-    oe_runmake -C ${S} all CFG_TEE_TA_LOG_LEVEL=0 CFG_TEE_CORE_LOG_LEVEL=0
-}
-
+LDFLAGS = ""
+CFLAGS += "--sysroot=${STAGING_DIR_HOST}"
+CXXFLAGS += "--sysroot=${STAGING_DIR_HOST}"
```

Comment:
Is those two need? I believe this is the default in toolchain arguments.

Reply:
I think so. I think the problem is the build files use `HOST_PREFIX` instead of using the tool variables where `--sysroot` resides. I will look into fixing that root issue.

Reply:
Looks non-trivial to rework the optee build files for this.